src/Entity/Server.php

```diff
@@ -59,7 +62,11 @@
  *     "endpoint",
  *     "debug_flag",
  *     "caching",
- *     "batching"
+ *     "batching",
+ *     "disable_introspection",
+ *     "depth",
+ *     "complexity",
+ *     "bypass_validation_token"
```

Comment:
we should also add the class properties on Server like we have with the other settings.